ports/esp32/machine_bitstream.c

```diff
@@ -90,6 +90,7 @@ static void IRAM_ATTR machine_bitstream_high_low_bitbang(mp_hal_pin_obj_t pin, u
     mp_hal_quiet_timing_exit(irq_state);
 }
 
+#if CONFIG_SOC_RMT_SUPPORTED
```

Comment:
```suggestion
#if SOC_RMT_SUPPORTED
```

(This is a nitpick, but the only reason the `CONFIG_` prefix copy exists is for KConfig.)